src/main/app/src/pages/HomePage.js

```diff
@@ -4,15 +4,21 @@ import { Link } from 'react-router-dom';
 import 'bulma/css/bulma.css';
 import '../css/home.css';
 
+// Web Servlet links.
```

Comment:
Is there a way to store these web servlet links so they can be shared across classes?

Reply:
We moved it to App.js so all components can have access to it in a later pull request.